src/type.ts

```diff
@@ -47,6 +47,9 @@ export type GetAppsResponse = {
 export interface ShowLocationProps {
   latitude: number | string;
```

Comment:
I'm interested in the address property only for my use-case. 
Should `latitude` & `longitude` be made optional?
```suggestion
  latitude?: number | string;
  longitude?: number | string;
```

Reply:
jerone, I've made the change. If you see anything else, please let me know.

Reply:
The example still shows empty strings.

Reply:
Thank you. Modified it.